zfit/models/dist_tfp.py

```diff
@@ -633,3 +633,76 @@ class ChiSquaredPDFRepr(BasePDFRepr):
     hs3_type: Literal["ChiSquared"] = Field("ChiSquared", alias="type")
     x: SpaceRepr
     ndof: Serializer.types.ParamTypeDiscriminated
+
+
+class StudentT(WrapDistribution, SerializableMixin):
+    _N_OBS = 1
+
+    def __init__(
+        self,
+        ndof: ztyping.ParamTypeInput,
+        mu: ztyping.ParamTypeInput,
+        sigma: ztyping.ParamTypeInput,
+        obs: ztyping.ObsTypeInput,
+        extended: ExtendedInputType = None,
+        norm: NormInputType = None,
+        name: str = "StudentT",
+    ):
+        """StudentT distribution for ndof degrees of freedom.
+
+        The StudentT shape for `d` degrees of freedom is defined as
+
+        .. math::
+
+            f(x \\mid d, \\mu, \\sigma) = \\left(1 + \\frac{1}{d} \\left(\\frac{x - \\mu}{\\sigma}\\right)^2\\right)^{-\\frac{d+1}{2}} / Z
+
+        with the normalization over [-inf, inf] of
+
+        .. math::
+
+            Z = \\frac{\\Gamma(\\frac{d+1}{2})}{\\sqrt{\\pi d} \\Gamma(\\frac{d}{2})}
+
+        The normalization changes for different normalization ranges
+
+        Args:
+            ndof: Number of degrees of freedom
+            mu: Mean of the distribution
+            sigma: Scale of the distribution
+            obs: Observables and normalization range the pdf is defined in
+            extended: |@doc:pdf.init.extended| The overall yield of the PDF.
+               If this is parameter-like, it will be used as the yield,
+               the expected number of events, and the PDF will be extended.
+               An extended PDF has additional functionality, such as the
+               ``ext_*`` methods and the ``counts`` (for binned PDFs). |@docend:pdf.init.extended|
+            norm: |@doc:pdf.init.norm| Normalization of the PDF.
+               By default, this is the same as the default space of the PDF. |@docend:pdf.init.norm|
+            name: |@doc:model.init.name| Human-readable name
+               or label of
+               the PDF for better identification.
+               Has no programmatical functional purpose as identification. |@docend:model.init.name|
+        """
+        ndof, mu, sigma = self._check_input_params(ndof, mu, sigma)
+        params = OrderedDict((("ndof", ndof), ("mu", mu), ("sigma", sigma)))
```

Comment:
```suggestion
        params = dict(ndof=ndof, mu=mu, sigma=sigma)
```
as dicts are insertion ordered, this should be all that is needed (I am also refactoring that for others)